src/main/scala/com/github/timgent/dataflare/checks/metrics/SingleMetricCheck.scala

```diff
@@ -92,9 +135,11 @@ object SingleMetricCheck {
     * @tparam MV
     * @return
     */
-  def sumValueCheck[MV <: NumericMetricValue: MetricValueConstructor]
-  (threshold: AbsoluteThreshold[MV#T], onColumn: String, filter: MetricFilter = MetricFilter.noFilter)
-  : SingleMetricCheck[MV] =
+  def sumValueCheck[MV <: NumericMetricValue: MetricValueConstructor](
+      threshold: AbsoluteThreshold[MV#T],
+      onColumn: String,
+      filter: MetricFilter = MetricFilter.noFilter
+  ): SingleMetricCheck[MV] =
     thresholdBasedCheck[MV](SumValuesMetric(onColumn, filter), "MinValueCheck", threshold)
```

Comment:
We can make the description as SumValueCheck. Looks like typo from my end

Reply:
Done